src/pages/CustomerInvoicesPage.js

```diff
@@ -144,7 +150,7 @@ export class CustomerInvoicesPage extends React.Component {
         return {
           type: 'checkable',
           icon: 'md-remove-circle',
-          isDisabled: invoice.isFinalised,
+          isDisabled: invoice.isFinalised || invoice.isLinkedToRequisition,
```

Comment:
The invoice should have a title, from requisition {serialNumber}. And when you press it the invoice, will navigate to requisition. Yes more verbose to the user would be nice!